modules/wmco-prerequisites.adoc

```diff
@@ -7,8 +7,37 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
-[id="wmco-prerequisites-supported-6.0.0_{context}"]
-== WMCO 7 supported platforms and Windows Server versions
+[id="wmco-prerequisites-supported-7.2.0_{context}"]
+== WMCO 7.2.x supported platforms and Windows Server versions
+
+The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 7.2.0 and 7.2.1, based on the applicable platform. Windows Server versions not listed are not supported and attempting to use them will cause errors. To prevent these errors, use only an appropriate version for your platform.
+
+[cols="3,7",options="header"]
+|===
+|Platform
+|Supported Windows Server version
```

Comment:
Consider making every cell under this heading have an unordered list (even when it's a single item) for more consistency/scannability